### ISSUE TITLE
Bug - Can't Handle Paths With Spaces In Their Names

### ISSUE THREAD

**Issue by drPeidos** [CosmicScale/PSBBN-Definitive-English-Patch#1]
Hi.
This should be an easy fix.
I use spaces to name my folders.
The 1st script didn't have any problems.

The 2nd script didn't like that:
`tee: /media/ponto/HGST: Permissão recusada
tee: 'raid/J,C&C/PS2': Ficheiro ou pasta inexistente
tee: Linux/PSBBN-installer.log: Ficheiro ou pasta inexistente

tee: /media/ponto/HGST: Permissão recusada
tee: 'raid/J,C&C/PS2': Ficheiro ou pasta inexistente
tee: Linux/PSBBN-installer.log: Ficheiro ou pasta inexistente
MODEL                  NAME          SIZE LABEL         MOUNTPOINT
HUS726060AL5210        /dev/sda      5,5T Azul:0        
                       └─/dev/md0    5,5T HGST raid     /media/ponto/HGST raid
HUS726060AL5210        /dev/sdb      5,5T Azul:0        
                       └─/dev/md0    5,5T HGST raid     /media/ponto/HGST raid
WDC WD5000LPCX-16VHAT1 /dev/sdc    465,8G               
                       ├─/dev/sdc1 465,8G Linux-WD500GB /
                       └─/dev/sdc2     1M 1MB-WD500GB   
WDC WD10EZEX-08WN4A0   /dev/sdd    931,5G               
                       └─/dev/sdd1 931,5G WD1TB         
tee: /media/ponto/HGST: Permissão recusada
tee: 'raid/J,C&C/PS2': Ficheiro ou pasta inexistente
tee: Linux/PSBBN-installer.log: Ficheiro ou pasta inexistente

Choose your PS2 HDD from the list above (e.g., /dev/sdx): `

Have not tested the 3rd script yet.

To solve this I think you just have to place all paths inside "".

Thanks.

EDIT: My Linux is in Portuguese:
**Permissão recusada** = _Access not granted, or something like that_
**Ficheiro ou pasta inexistente** = _File or folder doesn't exist, or something similar_
Basically the script is dividing the path in every space it finds and it's complaining that path is incorrect.

**Comment by CosmicScale** [comment]
Can you give me the full log file so I can see what is going on? Thanks

**Comment by drPeidos** [comment]
Hi.

In the terminal, when I run the 2nd script, this is what it shows:
`
See the attached "2nd script terminal.txt" file.
`
I close the terminal next, I don't tell it which drive to use.

I don't see any log file anywhere.
I stopped the 2nd script when it asked me which /dev/sdX should be used, I've never gone further, so maybe that's the reason why I don't see any log file.
I do have a lot of files with the same contents, here's all the files that are inside the root of the project:
`
See the attached "ls.txt" file.
`
I have attached those files.
[files.zip](https://github.com/user-attachments/files/18142238/files.zip)

**Comment by drPeidos** [comment]
[2nd script terminal.txt](https://github.com/user-attachments/files/18142289/2nd.script.terminal.txt)
[ls.txt](https://github.com/user-attachments/files/18142290/ls.txt)

**Comment by drPeidos** [comment]
I now added all the "" that I think are required on the 2nd script (see the attachment), but I'm not sure if everything will work like this.
[02-PSBBN-Installer.sh.zip](https://github.com/user-attachments/files/18142425/02-PSBBN-Installer.sh.zip)
With this modified 2nd script the log file is generated and the files I mentioned in my previous message are not generated.
[PSBBN-installer.log](https://github.com/user-attachments/files/18142426/PSBBN-installer.log)

**Comment by drPeidos** [comment]
[03-Game-Installer.sh.zip](https://github.com/user-attachments/files/18142493/03-Game-Installer.sh.zip)
I've implemented the same corrections to the 3rd script, but have not run it yet.

**Comment by CosmicScale** [comment]
It's a bit simpler than that. TOOLKIT_PATH should be declared in double quotes:
`TOOLKIT_PATH="$(pwd)"`

Every mention of TOOLKIT_PATH should also be in double quotes for example:
`"${TOOLKIT_PATH}"/helper/HDL\ Dump.elf`

Also `03-Game-Installer.sh`:
`MISSING_ART="${TOOLKIT_PATH}/missing-art.log"`

I will implement that tomorrow. Thanks for pointing out the problem.

**Comment by drPeidos** [comment]
Thanks for explaining the best way to correct this.
No rush on fixing this issue.
Thanks.

**Comment by CosmicScale** [comment]
This should now be fixed in [Commit 60c73a1](https://github.com/CosmicScale/PSBBN-Definitive-English-Patch/commit/60c73a13b794bda73786668c964cfa0a48c2d488#diff-e130b749364ea31f4e1e130881373ab9d3eb48eb8c61f6a889e41f0e64944c00R190)
